CHANGES.rst

```diff
@@ -1,3 +1,8 @@
+0.9.0 (unreleased)
```

Comment:
What happened to 0.8.0 change log? 👀

Reply:
It doesn't look like one was ever added.

Reply:
@zacharyburnett ?

Reply:
oh! I knew there was something I forgot

Reply:
OK opened https://github.com/spacetelescope/ci_watson/issues/75